src/jdk.jconsole/share/classes/sun/tools/jconsole/resources/messages_de.properties

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2012, 2024, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Same as my comment in src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTYResources_de.java